docs/install/index.md

```diff
@@ -1797,14 +1797,15 @@ install.packages("mxnet")
 
 <div class="gpu">
 
-You can [build MXNet-R from source](windows_setup.html#install-the-mxnet-package-for-r), or you can use a pre-built binary:
+You can [build MXNet-R from source](windows_setup.html#install-mxnet-package-for-r), or you can use a pre-built binary:
 
 ```r
 cran <- getOption("repos")
-cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU"
+cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU/cuX"
```

Comment:
suggest to write 

```
cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU/cu80"
```

as a concrete example, and then mention replacing `cu80` with `cu90`, `cu91`, `cu92` also works.

Reply:
@hetong007 fixed as suggested by you.